grpc/src/main/java/io/stargate/grpc/service/ExceptionHandler.java

```diff
@@ -94,9 +120,11 @@ private void handlePersistenceException(PersistenceException pe) {
         handleUnavailable((UnavailableException) pe);
         break;
       case OVERLOADED:
+        LOG.error(getErrorMessage(Status.RESOURCE_EXHAUSTED, pe));
```

Comment:
I am not sure we should log this one, it can occur when the client does not have a proper request rate limit setup..

Reply:
Hmmh. I can see how this could be problematic.

Let's remove logging here: we still get response code metrics (503 or something).